.github/workflows/charterafrica-deploy-prod.yml

```diff
@@ -11,9 +11,9 @@ concurrency:
 
 env:
   DOKKU_REMOTE_BRANCH: "master"
-  DOKKU_REMOTE_URL: "ssh://dokku@charter.prod.codeforafrica.org/charterafrica"
+  DOKKU_REMOTE_URL: "ssh://dokku@ui-1.prod.codeforafrica.org/codeforafrica-ui"
```

Comment:
Why `codeforafrica-ui`? The app should be named `charteafrica-ui-staging`, `charterafrica-ui-something`.

`codeforafrica-ui` reminds me of https://cfa.dev.codeforafrica.org/